src/components/shared/header-component/Header/SearchModal/search-modal.scss

```diff
@@ -1,10 +1,10 @@
 //invisible header in search modal in responsive
-.pgn__modal-header {
-  display: none;
-}
-.pgn__modal-close-container {
-  display: none;
-}
+// .pgn__modal-header {
+//   display: none;
+// }
+// .pgn__modal-close-container {
+//   display: none;
+// }
```

Comment:
remove this commented css selectors or if these code is needed in the future and there is uncertainty please mention:

       // This code is commented out because..... and will be used in the future

Reply:
Done